README.md

```diff
@@ -45,6 +47,14 @@ cd sensu-docs-site ; git submodule update --init --recursive
 This project uses [Yarn](https://yarnpkg.com/) to manage dependencies and the build process. For installation
 on installing yarn, [view their documentation](https://yarnpkg.com/lang/en/docs/install/).
 
+#### Installing Hugo
```

Comment:
Installing `hugo` is now handled automatically when running `yarn` or `yarn server`, so we shouldn't need these instructions any longer.